OTDefaultAudioDevice.h

```diff
@@ -7,33 +7,49 @@
 #import <Foundation/Foundation.h>
 #import <OpenTok/OpenTok.h>
 
-#define kMixerInputBusCount 2
-#define kOutputBus 0
-#define kInputBus 1
-
 #define AUDIO_DEVICE_HEADSET     @"AudioSessionManagerDevice_Headset"
 #define AUDIO_DEVICE_BLUETOOTH   @"AudioSessionManagerDevice_Bluetooth"
 #define AUDIO_DEVICE_SPEAKER     @"AudioSessionManagerDevice_Speaker"
 
 @interface OTDefaultAudioDevice : NSObject <OTAudioDevice>
-{
-  AudioStreamBasicDescription stream_format;
-}
 
 /**
- * Audio device lifecycle should live for the duration of the process, and
- * needs to be set before OTSession is initialized.
- *
- * It is not recommended to initialize unique audio device instances.
```

Comment:
So the problem with the previous ot network test implementation was that it was trying to call some methods that were not actually defined, so there was a mismatch somewhere along the way.

It looks like we never initialize a new OTDefaultAudioDevice instance, here is the code that is accessing it:
`OTDefaultAudioDevice *dev = (__bridge OTDefaultAudioDevice*) ref_con;`

@k7k0 Can we get your opinion?